apps/cowswap-frontend/.env

```diff
@@ -17,7 +17,7 @@ REACT_APP_FULL_APP_DATA_DEVELOPMENT='{"version":"0.7.0","appCode":"CoW Swap","en
 REACT_APP_FULL_APP_DATA_LOCAL='{"version":"0.7.0","appCode":"CoW Swap","environment":"local","metadata":{}}'
 
 # INFURA KEY
-REACT_APP_INFURA_KEY=586e7e6b7c7e437aa41f5da496a749f5
+REACT_APP_INFURA_KEY=2af29cd5ac554ae3b8d991afe1ba4b7d
```

Comment:
One thing that I'd like to raise is that, `REACT_APP_INFURA_KEY` is **required** and the app will fail to start when it's not set:

https://github.com/cowprotocol/cowswap/blob/0bf6c4eb2d940682abb242571003d03afd538a4c/libs/common-const/src/networks.ts#L23-L22

But it's not used at all if the other keys are set, which they are by default:

https://github.com/cowprotocol/cowswap/blob/0bf6c4eb2d940682abb242571003d03afd538a4c/libs/common-const/src/networks.ts#L17-L19

So I suggest to:
1. Make it a requirement **only if** one of the other env vars are missing
2. Remove it from `.env` files as it's redundant, and leads to confusion

I mention that because locally I changed only `REACT_APP_INFURA_KEY` in my `.env.local` and couldn't figure out why it was not being picked up.